pkgs/development/libraries/libbladeRF/default.nix

```diff
@@ -50,6 +59,11 @@ in stdenv.mkDerivation {
     "-DBLADERF_GROUP=bladerf"
   ];
 
+  # The default of libbladeRF, -Werror, is meant for development where there is
+  # control over the compiler, but in nixpkgs we want to update gcc without
+  # breaking half the packages due to warnings.
+  NIX_CFLAGS_COMPILE = [ "-Wno-error" ];
```

Comment:
Do we need this if we have the patch?

Reply:
No, we don't need this; I picked this up on the suggestion of @Mac92 over at #108585. I don't have a strong opinion on that matter.

Reply:
We should do one of the things: the patch or removing -Werror.

Reply:
I see. Please you or somebody else make the call, I'm not experienced enough with our conventions.

Reply:
Because the patch is upstream I would prefer it.

Reply:
Okay, thanks, I'll force-push accordingly in a second. :-)